### DISCUSSION TITLE
What is _initialAnswer

### DISCUSSION THREAD

**Discussion by Robschuh** [Cyfrin/foundry-full-course-cu#1860] [General]
Hi there, I do not understand why to initialize **_initialAnswer** with  2000e8 (200.000 000 000). Why not a different number, or cero?

  ```
constructor(uint8 _decimals, int256 _initialAnswer) {
        decimals = _decimals;
        updateAnswer(_initialAnswer);
    }

```

**Comment by urosognjenovic** [comment]
Initial answer is the result you'll get when you fetch the price from your MockV3Aggregator contract. You can pass any value actually. 2000 is close to the actual real price of Ether.

**Comment by EngrPips** [comment]
Hello @Robschuh, `_initialAnswer` in the context of this contract is the actual amount in `USD` that your token is worth. So if you initialize it to `2000e8`, that means one of your token is worth `2000e8`, which is `2000USD` because the `e8` is just the decimal place. So you can really initialize it to anything, but to make the code base real, we initialize it to `2000e8` as that is close to the price of ETH.

**Reply by r0bchain** [reply]
Yeah this is what I supposed, that you can pass any amount of USD.
Thanks guys!

**Reply by EngrPips** [reply]
You are welcome.
